tempesta_fw/cache.c

```diff
@@ -158,29 +158,27 @@ typedef struct {
 	TDB		*db;
 } CaNode;
 
-static CaNode c_nodes[MAX_NUMNODES];
-
-static struct task_struct *cache_mgr_thr;
-static DEFINE_PER_CPU(TfwWorkTasklet, cache_wq);
-
-static TfwStr g_crlf = { .ptr = S_CRLF, .len = SLEN(S_CRLF) };
+	static CaNode c_nodes[MAX_NUMNODES];
+	static struct task_struct *cache_mgr_thr;
+	static DEFINE_PER_CPU(TfwWorkTasklet, cache_wq);
+	static TfwStr g_crlf = { .data = S_CRLF, .len = SLEN(S_CRLF) };
```

Comment:
Wrong formatting. A definition of a global variable must begin at the start of a line.